src/components/index/page-sections/IntroductionSection.jsx

```diff
@@ -5,16 +5,23 @@ import { DesktopQuery } from '../../../util/helpers';
 
 function IntroductionSection({ children }) {
   return (
-    <div className="relative overflow-hidden mb-40">
+    <div className="relative overflow-hidden items-center justify-center h-screen mb-40">
```

Comment:
Da klein flex gesetzt ist haben items-center und justify-center hier keinen Effekt, oder? Besser evtl w-full oder w-auto (je nachdem wies aussehen soll)

Reply:
Das hatte ich gar nicht bearbeitet - aber ich denke auch, in der Index Page war kein flex gesetzt